operator/src/main/java/oracle/kubernetes/operator/EventConstants.java

```diff
@@ -29,9 +29,9 @@
   String DOMAIN_PROCESSING_FAILED_PATTERN
       = "Failed to complete processing domain resource %s due to: %s, the processing will be retried if needed";
```

Comment:
"if needed" seems vague

when isn't it needed?
how many retry attempts will there be?

Reply:
I've been informed in slack that current behaviors are not well known and are subject to change.
If the above comment can't easily be addressed, can you file a follow on JIRA that (a) references the pending retry work, this retry work, and this pull, (b) describes enhancing this particular error message, and finally (c) asks to enhance  events so that they include the expected retry interval?

Reply:
Apparently this task depends on OWLS-84438. It seems premature to work on this. Based on a discussion offline, we decided to hold on proceeding this PR, and resume working on this after OWLS-84438 is resolved. I am closing this PR.